app/cafe.py

```diff
@@ -0,0 +1,18 @@
+from datetime import date
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+from app.errors import NotVaccinatedError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+        if visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("All friends should be vaccinated")
+        if visitor["wearing_a_mask"] is False:
```

Comment:
u don't need to use id comparison in this case
```suggestion
        if not visitor["wearing_a_mask"]:
```